tutorial/markdown/java/spring-boot.md

```diff
@@ -53,172 +91,494 @@ mvn package
+`AirlineController.java`
+This class defines the RESTful API endpoints for managing airlines. It handles HTTP requests for creating, updating, deleting, and retrieving airlines. It also provides endpoints for listing airlines by various criteria.
+
+An example of the pattern for the `GET` mapping is shown below.For the full code, see the [AirlineController.java](https://github.com/couchbase-examples/java-springboot-quickstart/blob/main/src/main/java/org/couchbase/quickstart/springboot/controllers/AirlineController.java).
 
 ```java
-public Profile getProfile() {
-  return new Profile(
-    UUID.randomUUID().toString(),
-    firstName,
-    lastName,
-    email,
-    password);
+@RestController
+@RequestMapping("/api/v1/airline")
+@Slf4j
+public class AirlineController {
+
+   private final AirlineService airlineService;
+
+    public AirlineController(AirlineService airlineService) {
+        this.airlineService = airlineService;
+    }
+
+    // Error messages
+    private static final String INTERNAL_SERVER_ERROR = "Internal Server Error";
+    private static final String DOCUMENT_NOT_FOUND = "Document Not Found";
+
+    @GetMapping("/{id}")
+    @Operation(summary = "Get an airline by ID")
+    @Description(value = "Get Airline by specified ID.\n\nThis provides an example of using Key Value operations in Couchbase to retrieve a document with a specified ID. \n\n Code: [`controllers/AirlineController.java`](https://github.com/couchbase-examples/java-springboot-quickstart/blob/master/src/main/java/org/couchbase/quickstart/springboot/controllers/AirlineController.java) \n File: `AirlineController.java` \n Method: `getAirline`")
+    public ResponseEntity<Airline> getAirline(@PathVariable String id) {
+        try {
+            Airline airline = airlineService.getAirlineById(id);
+            if (airline != null) {
+                return new ResponseEntity<>(airline, HttpStatus.OK);
+            } else {
+                log.error(DOCUMENT_NOT_FOUND + ": " + id);
+                return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+            }
+        } catch (DocumentNotFoundException e) {
+            log.error(DOCUMENT_NOT_FOUND + ": " + id);
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        } catch (Exception e) {
+            log.error(INTERNAL_SERVER_ERROR + ": " + e.getMessage());
+            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
 }
+
 ```
 
-> *from Post method of models/ProfileRequest.java*
+### Service
 
-Rather than saving the password in the account object as plain text, we hash it with [Bcrypt](https://github.com/jeremyh/jBCrypt) in the setter of the Profile object:
+`AirlineServiceImpl.java`
```

Comment:
You need to split this code into relevant sections below

Reply:
Have removed unnecessary sections and attached only the relevant parts